java/pom.xml

```diff
@@ -21,12 +21,28 @@
             <name>Backup Repository</name>
             <url>https://repo2.maven.org/maven2/</url>
         </repository>
-
         <repository>
-            <id>snapshot-repository</id>
-            <name>Hazelcast Snapshot Repository</name>
-            <url>https://oss.sonatype.org/content/repositories/snapshots</url>
-        </repository>
+            <id>snapshot-internal</id>
+            <name>Hazelcast Internal Snapshots</name>
+            <url>https://repository.hazelcast.com/snapshot-internal/</url>
+            <releases>
+                <enabled>false</enabled>
+            </releases>
+            <snapshots>
+                <enabled>true</enabled>
+            </snapshots>
+          </repository>
+          <repository>
+            <id>releaese-internal</id>
+            <name>Hazelcast Internal</name>
+            <url>https://repository.hazelcast.com/release</url>
+            <releases>
+                <enabled>true</enabled>
+            </releases>
+            <snapshots>
+                <enabled>false</enabled>
+            </snapshots>
+          </repository>
```

Comment:
For enabling `5.5.0` + `./build` + test. Checked it works fine.

Reply:
repo element `<id>snapshot-internal</id>` is unnecessary for public access (though harmless to keep)